demo/config.js

```diff
@@ -271,7 +271,9 @@ shakaDemo.Config = class {
             /* canBeDecimal= */ true)
         .addNumberInput_(MessageIds.SLOW_HALF_LIFE,
             'abr.advanced.slowHalfLife',
-            /* canBeDecimal= */ true);
+            /* canBeDecimal= */ true)
+        .addBoolInput_(MessageIds.QUALITY_RESTRICTIONS_ELEMENT_SIZE,
+            'abr.qualityRestrictionsElementSize');
```

Comment:
I think this could use a better name.  I initially thought it meant passing a size, not a boolean.

How about `abr.restrictToElementSize` ?

Reply:
Done!